simulation-model/simulation-model_instance.json

```diff
@@ -0,0 +1,194 @@
+        "skos": "http://www.w3.org/2004/02/skos/core#",
+        "georeference": "https://github.com/GAIA-X4PLC-AAD/ontology-management-base/tree/main/georeference/"
+    },
+    "@id": "did:web:registry.gaia-x.eu:SimulationModel:2Jt36BA9ODxKZJ5yWRtF4BLDUkuvFL59IPra",
+    "@type": "simulation-model:SimulationModel",
+    "simulation-model:general": {
+        "@type": "general:General",
+        "general:description": {
+            "@type": "general:Description",
+            "general:name": {
+                "@value": "Simulation Model Example",
+                "@type": "xsd:string"
+            },
+            "general:description": {
+                "@value": "A sample description of a none existing simulation model.",
+                "@type": "xsd:string"
+            }
+        },
+        "general:data": {
+            "@type": "general:Data",
+            "general:size": {
+                "@value": "1.0",
+                "@type": "xsd:float"
+            },
+            "general:contractId": {
+                "@value": "1234567890",
+                "@type": "xsd:string"
+            },
+            "general:recordingTime": {
+                "@value": "2024-06-26T09:00:00",
+                "@type": "xsd:dateTime"
+            }
+        },
+        "general:links": {
+            "@type": "general:Links",
+            "general:data": {
+                "@type": "general:Link",
+                "general:url": {
+                    "@value": "https://github.com/openMSL/sl-1-1-reflection-based-radar-object-model",
+                    "@type": "xsd:anyURI"
+                },
+                "general:type": "Asset"
+            },
+            "general:media": {
+                "@type": "general:Link",
+                "general:url": {
+                    "@value": "https://inavlid.link.com/media",
+                    "@type": "xsd:anyURI"
+                },
+                "general:type": "Video"
+            }
+        },
+        "general:bundleData": {
+            "@type": "general:BundleData",
+            "general:requiredData": {
+                "@type": "general:Link",
+                "general:url": {
+                    "@value": "did:web:registry.gaia-x.eu:OSITrace:T9rkdS19poUrbbAEk3rXMqZ3hzVezwDrrxaf",
+                    "@type": "xsd:anyURI"
+                },
+                "general:type": "Asset"
+            },
+            "general:relatedData": {
+                "@type": "general:Link",
+                "general:url": {
+                    "@value": "did:web:registry.gaia-x.eu:OSITrace:T9rkdS19poUrbbAEk3rXMqZ3hzVezwDrrxaf",
+                    "@type": "xsd:anyURI"
+                },
+                "general:type": "Asset"
+            }
+        }
+    },
+    "simulation-model:format": {
+        "@type": "simulation-model:Format",
+        "https://github.com/GAIA-X4PLC-AAD/ontology-management-base/tree/main/simulation-model/model.implementation.format": {
```

Comment:
I'm not sure if these links resolve properly. The IRI/namespace is already defined in the @context, so these links could probably be omitted?

Reply:
Yeah I'm also not sure about this. This where directly generated from the wizard but in the other examples these are not prefixed. I don't know why. I think I should delete them. However, @robertschubert do you have any idea why these where included and if we should keep them (maybe these are now inserted because I used the `.` in the `sh:path`'s)?